src/Core/TileMesh.js

```diff
@@ -237,4 +237,31 @@ TileMesh.prototype.getZoomForLayer = function getZoomForLayer(layer) {
     }
 };
 
+/**
+ * Search for a common ancestor between this tile and another one. It goes
+ * through parents on each side until one is found.
+ *
+ * @param {TileMesh} tile
+ *
+ * @return {TileMesh} the resulting common ancestor
+ */
+TileMesh.prototype.findCommonAncestor = function findCommonAncestor(tile) {
```

Comment:
is that [this code](https://github.com/iTowns/itowns/pull/719/files#diff-6b60bae2240ab556c955b4eb0a07a12eR189)  is similar in ancestor search with different criteria (id and name)

Reply:
It is similar, but I don't see how it could be mutualized: pointcloud is a very specific usecase.